core/admin.py

```diff
@@ -5,6 +5,8 @@
 from . import models
 from . import tasks
 
+import os
```

Comment:
We don't (yet) have a style rule for this, but usually we put imports from external and system packages before local packages.  Can move this to just below the django imports?

Reply:
yes